core/assets/bundles/bundle_pt_BR.properties

```
@@ -2089,41 +2089,41 @@ unit.evoke.description = Constrói estruturas para defender o Bastião do Núcle
 
-lst.read = Read a number from a linked memory cell.
-lst.write = Write a number to a linked memory cell.
-lst.print = Add text to the print buffer.\nDoes not display anything until [accent]Print Flush[] is used.
-lst.draw = Add an operation to the drawing buffer.\nDoes not display anything until [accent]Draw Flush[] is used.
-lst.drawflush = Flush queued [accent]Draw[] operations to a display.
-lst.printflush = Flush queued [accent]Print[] operations to a message block.
-lst.getlink = Get a processor link by index. Starts at 0.
-lst.control = Control a building.
-lst.radar = Locate units around a building with range.
-lst.sensor = Get data from a building or unit.
-lst.set = Set a variable.
-lst.operation = Perform an operation on 1-2 variables.
-lst.end = Jump to the top of the instruction stack.
-lst.wait = Wait a certain number of seconds.
-lst.stop = Halt execution of this processor.
-lst.lookup = Look up an item/liquid/unit/block type by ID.\nTotal counts of each type can be accessed with:\n[accent]@unitCount[] / [accent]@itemCount[] / [accent]@liquidCount[] / [accent]@blockCount[]
-lst.jump = Conditionally jump to another statement.
-lst.unitbind = Bind to the next unit of a type, and store it in [accent]@unit[].
-lst.unitcontrol = Control the currently bound unit.
-lst.unitradar = Locate units around the currently bound unit.
-lst.unitlocate = Locate a specific type of position/building anywhere on the map.\nRequires a bound unit.
-lst.getblock = Get tile data at any location.
-lst.setblock = Set tile data at any location.
-lst.spawnunit = Spawn unit at a location.
-lst.applystatus = Apply or clear a status effect from a unit.
-lst.spawnwave = Spawn a wave.
-lst.explosion = Create an explosion at a location.
-lst.setrate = Set processor execution speed in instructions/tick.
-lst.fetch = Lookup units, cores, players or buildings by index.\nIndices start at 0 and end at their returned count.
-lst.packcolor = Pack [0, 1] RGBA components into a single number for drawing or rule-setting.
-lst.setrule = Set a game rule.
-lst.flushmessage = Display a message on the screen from the text buffer.\nWill wait until the previous message finishes.
-lst.cutscene = Manipulate the player camera.
-lst.setflag = Set a global flag that can be read by all processors.
-lst.getflag = Check if a global flag is set.
+lst.read = Lê um numero de uma célula de memoria conectada.
+lst.write = Escreve um numero em uma célula de memória conectada.
+lst.print = Adiciona texto ao buffer de impressão.\nNão vai exibir nada até o [accent]Print Flush[] ser usado.
+lst.draw = Adiciona uma operação ao buffer de desenho.\nNão vai exibir nada até o [accent]Draw Flush[] ser usado.
+lst.drawflush = Libera operações de [accent]Draw[] em fila para ser exibida.
+lst.printflush = Libera operações de [accent]Print[] em fila para um bloco de mensagem.
+lst.getlink = Pega uma conexão do processador pelo index. Começa no 0.
```

Change "index" to "indice" so it matches the translation at line 2120.
```suggestion
lst.getlink = Pega uma conexão do processador pelo indice. Começa no 0.
```